Extensions/FileSystem/JsExtension.js

```diff
@@ -59,6 +59,24 @@ module.exports = {
       )
       .setFunctionName("gdjs.fileSystem.makeDirectory");
 
+    extension
+      .addAction(
+        "SaveTextToFile",
```

Comment:
I'm still not sure about if we should use Text or String everywhere in the user visible fields.
But internally, better stick to string as it's the "programming valid" word for this.

So `SaveStringToFile` and `gdjs.fileSystem.saveStringToFile` :)

Reply:
I initially used `SaveStringToFile` as name until I saw that the storage extension was using the term `Text` in the visible fields. But I guess splitting it up like you suggested is the best way.

Reply:
For internal conditions/actions name and for expression, it's better to always stick with string - at least if it was not done before, we should now do it :) For displaying to the user, we talked about it here: https://github.com/4ian/GDevelop/issues/846

I still don't have a final choice in mind. Text sounds easy to understand for everyone, but string is the correct programming term.